central/processlisteningonport/datastore/datastore_impl.go

```diff
@@ -481,6 +487,7 @@ func sortByCloseTimestamp(values []*storage.ProcessListeningOnPortFromSensor) {
 
 func addNewPLOP(plopObjects []*storage.ProcessListeningOnPortStorage,
 	indicatorID string,
+	clusterID string,
```

Comment:
Unless I missed a place, you don't need to pass the clusterID here because you added it to the PLOP before calling this.  So you could leave the signature here as it was and then `ClusterID: value.ClusterID` below.

Reply:
It is needed, because sensor does not actually add the clusterID. I should probably remove clusterID from ProcessListeningOnPortFromSensor, as it is not used. I could modify the sensor code so that clusterID is set, but I don't think that is the best solution, as sensor could be on an older version than central and then clusterID would be null. Setting clusterID in central guarantees that it will be set.

Reply:
Before you call this method, you always set it in the `value` parameter.  That is why I was questioning if it was necessary or not.

Reply:
Oops. I thought you meant that I didn't need to pass `clusterID` to AddProcessListeningOnPort. I didn't look carefully.